Makefile

```diff
@@ -37,14 +37,20 @@ all: default
 # Platform #
 ############
 
+ifeq ($(VARIANT),test)
+  DEFINES   += INS_BASE58_ENCODE_TEST_FUNC
+  DEFINES   += INS_CBOR_DECODE_TEST_FUNC
+else
+  DEFINES   += INS_SIGN_TX_FUNC
+  DEFINES   += INS_SET_TX_FUNC
+  DEFINES   += INS_GET_PUBLIC_KEY_FUNC
+  DEFINES   += INS_GET_WALLET_INDEX_FUNC
+  DEFINES   += INS_GET_RND_PUB_KEY_FUNC
+endif
+
 DEFINES   += OS_IO_SEPROXYHAL IO_SEPROXYHAL_BUFFER_SIZE_B=128
 DEFINES   += HAVE_BAGL HAVE_PRINTF
-DEFINES   += INS_SIGN_TX_FUNC INS_SET_TX_FUNC
-DEFINES   += INS_GET_PUBLIC_KEY_FUNC
-#DEFINES   += INS_GET_WALLET_INDEX_FUNC
-#DEFINES   += INS_GET_RND_PUB_KEY_FUNC
-#DEFINES   += INS_HASH_FUNC
-#DEFINES   += INS_CBOR_DECODE_TEST_FUNC
+DEFINES   += INS_HASH_FUNC
```

Comment:
Move INS_HASH_FUNC into the test functions. I'll rename it at some point to INS_HASH_TEST_FUNC

Reply:
Boom, will do!